assets/src/dashboard/app/api/useStoryApi.js

```diff
@@ -193,6 +194,49 @@ const useStoryApi = (dataAdapter, { editStoryURL, wpApi }) => {
     },
     [wpApi, dataAdapter]
   );
+  const createStoryFromTemplate = useCallback(
+    async (template) => {
+      try {
+        const { createdBy, pages, title, version } = template;
+        const copyTemplateTitle = sprintf(
+          /* translators: %s: template title */
+          __('%s (Copy)', 'web-stories'),
+          title
+        );
+
+        const storyPropsToSave = await getStoryPropsToSave({
+          story: {
+            title: copyTemplateTitle,
+            status: 'draft',
+            date: new Date(),
+          },
+          pages,
+          metadata: {
+            publisher: {
+              name: createdBy,
+            },
+          },
+        });
+        const response = await dataAdapter.post(wpApi, {
+          data: {
+            ...storyPropsToSave,
+            story_data: {
+              pages,
+              version,
+              autoAdvance: true,
+              defaultPageDuration: 7,
+            },
+          },
+        });
+
+        return (window.location = `${editStoryURL}&post=${response.id}`);
```

Comment:
```suggestion
        return (window.location = addQueryArgs( editStoryURL, { post: response.id} ));
```

Reply:
nice! i didn't know this was a util in the edit-story side of things, thank you.